src/test/java/com/ihren/task2/exercise3/ItemMapperImplTest.java

```diff
@@ -0,0 +1,298 @@
+package com.ihren.task2.exercise3;
+
+import com.ihren.task2.exercise3.model.Element;
+import com.ihren.task2.exercise3.model.Item;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.then;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+
+@ExtendWith(MockitoExtension.class)
+class ItemMapperImplTest {
```

Comment:
According to logic branches. You have one branch for any method. So, you should have two test methods.